apel/common/message_schemas.py

```diff
@@ -0,0 +1,149 @@
+"""A file to store message schemas for JSON based messages"""
+
+IP_MSG_SCHEMA = {
```

Comment:
As a note to Future Greg, we probably want to version this contant in some sense, i.e. `IP_MSG_SCHEMA_V0.2`.